src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSink.java

```diff
@@ -258,4 +260,16 @@ private static void mergeInto(OutputStream out, Path directory, Configuration co
             fs.delete(part.getPath(), false);
         }
     }
+
+    protected static FileStatus[] getBamFragments( final Path directory, final FileSystem fs ) throws IOException {
+        final FileStatus[] parts = fs.globStatus(new Path(directory, "part-r-[0-9][0-9][0-9][0-9][0-9]*"));
```

Comment:
this should be based on the value of `mapreduce.output.basename` in case someone sets it on us